Detections/MultipleDataSources/SuspiciousAWSCLICommandExecution.yaml

```diff
@@ -0,0 +1,47 @@
+id: 8c2dc344-9352-4ca1-8863-b1b7a5e09e59
+name: Suspicious AWS CLI Command Execution
+description: |
+  'This detection focuses on identifying potentially suspicious activities involving the execution of AWS Command Line Interface (CLI) commands, particularly focusing on reconnaissance operations.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Reconnaissance
+relevantTechniques:
+  - T1596

```

Comment:
Not sure I agree with this technique mapping

Reply:
Update : 
relevantTechniques:
  - T1595
  - T1592.004
  - T1589.002
  - T1589.003
  - T1590
  - T1591
  - T1596